docs/src/migration.md

```diff
@@ -0,0 +1,275 @@
+# MetaGraphs.jl to MetaGraphsNext.jl
+
+This section is mainly targeted towards people intending to migrate from
+*MetaGraphs.jl* to *MetaGraphsNext.jl*. We present how one could implement the
+examples shown in *MetaGraphs.jl*'s
+[Example Usage](https://juliagraphs.org/MetaGraphs.jl/dev/#Example-Usage)
+section using *MetaGraphsNext.jl*.
+
+***
+
+###### Import packages
+
+```@example main
+using Graphs
+import MetaGraphs, MetaGraphsNext
+```
+
+***
+
+###### Create a metagraph based on a simplegraph, with optional default edge weight
+
+Using *MetaGraphs.jl*:
+```@example main
+_mg = MetaGraphs.MetaGraph(path_graph(5), 3.0)
+```
+
+Using *MetaGraphsNext.jl*:
+```@example main
+mg = MetaGraphsNext.MetaGraph(Graph(), default_weight = 3.0)
```

Comment:
Instead of doing each vertex one at a time, it might be nice to add a constructor to do it all at once. Ref #6?

Reply:
Yes! That constructor would be super cool to have. Perhaps I can find some time to implement it. I think I would need some assistance though.

Reply:
Ok, let me know if you have any questions.